SUCCESS\
assertion failed: p.sum() == 3
